pkg/ddevapp/task.go

```diff
@@ -78,12 +80,10 @@ func (c ExecHostTask) Execute() error {
 		bashPath = util.FindWindowsBashPath()
 	}
 
-	cmd := exec.Command(bashPath, "-c", c.exec)
-	var stdout bytes.Buffer
-	var stderr bytes.Buffer
-	cmd.Stdout = &stdout
-	cmd.Stderr = &stderr
-	err = cmd.Run()
+	args := []string{"-c"}
+	args = append(args, c.exec)
+
+	err = exec.RunInteractiveCommand(bashPath, args)
```

Comment:
Looks pretty elegant to me!

Reply:
Or this should also work
```
args := []string{
    "-c",
    c.exec,
}
```

Reply:
I did not know there is such a big difference for the two solutions, the second one safes 43 OpCodes so we should definitively go with it.
![grafik](https://user-images.githubusercontent.com/25326036/97102030-4e5a8400-16a2-11eb-9672-76a815d85606.png)